server/aws-lsp-identity/src/language-server/identityService.ts

```diff
@@ -42,6 +42,9 @@ export class IdentityService {
         if (!ssoToken) {
             // If no cached token and cannot start the login process, give up
             if (!options.loginOnInvalidToken) {
+                this.observability.logging.log(
+                    'SSO token not found an loginOnInvalidToken = false, returning no token.'
```

Comment:
Don't you want to know more info about the failure, like `source`?

Reply:
I left out shared config info entirely due to AppSec request not to leak PII, including names such as in paths.  These are things that can be added later if not considered unsecure and necessary.